tests/NAnt.Core/FrameworkInfoTest.cs

```diff
@@ -177,6 +177,7 @@ public class FrameworkInfoTest : BuildTestBase {
         }
 
         [Test]
+        [Ignore("Inconsistent namespaces for 'GetManifestResourceStream' method between .NET and Mono")]
```

Comment:
Why ignore this test?  Should probably use preprocessers to address the inconsistent namespace issue instead of just ignoring it.